jetty-server/src/main/java/org/eclipse/jetty/server/AbstractConnector.java

```diff
@@ -161,7 +164,18 @@
     private long _idleTimeout = 30000;
     private String _defaultProtocol;
     private ConnectionFactory _defaultConnectionFactory;
+    /* The name used to link up virtual host configuration to named connectors */
     private String _name;
+    /**
+     * The authority used to define the server uri authority fallback for the connection
+     * (see ServletRequest.getServerName(), and ServletRequest.getServerPort()).
+     */
+    private HostPort _serverAuthority;
+    /**
+     * The address used to define the local address for the connection
+     * (see ServletRequest.getLocalName(), ServletRequest.getLocalAddr(), and ServletRequest.getLocalPort()).
+     */
+    private SocketAddress _localAddress;
```

Comment:
I think it is sufficient to have javadoc on the associated getters/setters rather than on the private fields

Reply:
I'll move these to the `Connector` apidoc, as it's kinda useful information not represented there currently.

Reply:
Moved